src/ui/QuerySummary/QuerySummary.ts

```diff
@@ -294,10 +288,24 @@ export class QuerySummary extends Component {
     }
   }
 
+  private hideCustomNoResultsFoundPage() {
+    const showIfNoResultsElement = this.element.getElementsByClassName(SHOW_IF_NO_RESULTS);
+    if (showIfNoResultsElement.length > 0) {
```

Comment:
If you use `$$`, change that check to verify that it exists, it will only return one element (not an array)

Reply:
Good point!